test/concurrent/emit-fail-interrupt.cc

```diff
@@ -30,13 +30,13 @@ TYPED_TEST(ConcurrentTypedTest, EmitFailInterrupt) {
   auto e = [&]() {
     return Stream<int>()
                .interruptible()
-               .begin([](auto& k, Interrupt::Handler& handler) {
-                 handler.Install([&k]() {
+               .begin([](auto& k, auto& handler) {
+                 handler->Install([&k]() {
```

Comment:
```suggestion
                 CHECK(handler) << "Test expects interrupt to be registered";
                 handler->Install([&k]() {
```

Here and everywhere else!